api/policies/sessionAuth.js

```diff
@@ -1,8 +1,32 @@
-module.exports = function(req, res, next) {
-  req.session.authRedirectPath = undefined
+const GitHub = require('../services/GitHub');
+const utils = require('../utils');
+
+module.exports = function sessionAuth(req, res, next) {
+  req.session.authRedirectPath = undefined;
   if (req.session.authenticated) {
+    const lastAuthenticatedAt = req.session.authenticatedAt;
+    if (!lastAuthenticatedAt || utils.isPastAuthThreshold(lastAuthenticatedAt)) {
+      // check that the authenticated user is still a member of a valid GitHub org
+      return GitHub.validateUser(req.user.githubAccessToken)
```

Comment:
They are not, but when they log in they get one so all authenticated users should have one 👍